src/DotLiquid/Hash.cs

```diff
@@ -274,32 +275,32 @@ public int Count
 
         public object SyncRoot
         {
-            get { return ((IDictionary) _nestedDictionary).SyncRoot; }
+            get { return ((IDictionary)_nestedDictionary).SyncRoot; }
         }
 
         public bool IsSynchronized
         {
-            get { return ((IDictionary) _nestedDictionary).IsSynchronized; }
+            get { return ((IDictionary)_nestedDictionary).IsSynchronized; }
         }
 
         ICollection IDictionary.Values
         {
-            get { return ((IDictionary) _nestedDictionary).Values; }
+            get { return ((IDictionary)_nestedDictionary).Values; }
         }
 
         public bool IsReadOnly
         {
-            get { return ((IDictionary<string, object>) _nestedDictionary).IsReadOnly; }
+            get { return ((IDictionary<string, object>)_nestedDictionary).IsReadOnly; }
         }
 
         public bool IsFixedSize
         {
-            get { return ((IDictionary) _nestedDictionary).IsFixedSize; }
+            get { return ((IDictionary)_nestedDictionary).IsFixedSize; }
         }
 
         public bool ContainsKey(string key)
         {
-            return _nestedDictionary.ContainsKey(key);
+            return Contains(key);

```

Comment:
Needs research why we have two alias methods. I only changed and tested this one in my commit. Maybe we can mark the other obsolete?

Reply:
The Contains/ContainsKey methods are for different interfaces implemented by Hash:
- `System.Collections.Generic.IDictionary<string, object>` declares `ContainsKey(string)`
- `System.Collections.IDictionary` declares `Contains(object) `